components/infobox/wikis/rocketleague/infobox_team_custom.lua

```diff
@@ -91,7 +91,10 @@ function CustomTeam:addToLpdb(lpdbData, args)
 		lpdbData.logo = 'File:' .. args.image
 	end
 
-	lpdbData.extradata = { rating = Variables.varDefault('rating') }
+	lpdbData.extradata = {
+		rating = Variables.varDefault('rating'),
+		tier = string.lower(args.regionsubtier or '')
```

Comment:
```suggestion
		tier = string.lower(args.tieror '')
```

Reply:
adjusted (but with the space before the `or`